Doc/library/time.rst

```diff
@@ -256,6 +256,13 @@ Functions
    Like :func:`gmtime` but converts to local time.  If *secs* is not provided or
    :const:`None`, the current time as returned by :func:`.time` is used.  The dst
    flag is set to ``1`` when DST applies to the given time.
+   
```

Comment:
I think this has a lone space as well, but can't seem to suggest the edit for you.